packages/backend/src/storage/storage.ts

```diff
@@ -398,6 +399,16 @@ export class Storage {
       throw new Error(`Couldn't open file ${fileContent.path}. Error: ${e.message}`)
     }
 
+    let width = null
+    let height = null
+    try {
+      const fileDimensions = sizeOf(buffer)
+      width = fileDimensions.width
+      height = fileDimensions.height
+    } catch (e) {
+      log(`Can't read ${fileContent.path} dimensions`)
```

Comment:
I assume that catch block will execute in case uploaded file is not an image. I'd add this explanation to log message.

Reply:
Added. Generally I think we will get read of this log as soon as we implement handling other files because otherwise it would spam. I would leave it for now for debugging purposes.